### DISCUSSION TITLE
Araba tekerlek sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#131] [Zeka Sorusu]
Bir aracın önünde kullanılan tekerlek 63000 km, arkasında kullanılan tekerlek 72000 km gidebiliyor.
Bu araçta 5 tekerlek (biri yedek) olduğuna göre bu araç en fazla kaç km yol alabilir?

**Comment by Ersan0102** [comment]
63000 km gidebilir

**Reply by sonsuzus** [reply]
Zaten en az 63 bin kilometre gidebilir. En fazla götürecek hiç mi yöntem yok?

**Reply by Ersan0102** [reply]
Hocam birkaç yöntem denedim fakat hepsinde en fazla 63000 km gidebildi

**Reply by sonsuzus** [reply]
Zaten yedek lastiği sırayla değiştirsen 15 bin km fazladan gider.
